docs/workforce-management-framework/topics/con-raincatcher-core-getting-started.adoc

```diff
@@ -0,0 +1,143 @@
+[[id='raincatcher-core-getting-started-{chapter}']]
+= RainCatcher-core Getting Started
+
+Getting started with the Feedhenry RainCatcher Core Repository
+
+== Developing
+
+RainCatcher supports and is tested on Node.JS@4 and should run on latest LTS.
```

Comment:
what `Node.JS@4` means :)?? This will become outdated once we release. We need to target node.js 6TLS plus latest LTS